node-module-system/calculate.js

```diff
@@ -0,0 +1,20 @@
+import add from "./add.js";
+import subtract from "./subtract.js";
+import multiply from "./multiply.js";
+import divide from  "./divide.js";
+if (isNaN(+process.argv[2])) {
+  console.log(process.argv[2] + ' is not a number.');
+} else if (isNaN(process.argv[4])) {
+  console.log(process.argv[4] + ' is not a number.');
+}  else if ((typeof (+process.argv[2]) == 'number') && (typeof (+process.argv[4]) == 'number')) {
```

Comment:
This extra check is not necessary since `isNan` will give you the same information.